posix.mak

```diff
@@ -375,6 +377,16 @@ unittest/%.run : $(ROOT)/unittest/test_runner
 %.test : $(LIB)
 	$(MAKE) -f $(MAKEFILE) $(addsuffix .test,$(patsubst %.d,%,$(wildcard $*/*)))
 
+%.debug_with_debugger : %.d $(LIB)
+	$(DMD) $(DFLAGS) -main -unittest $(LIB) -defaultlib= -debuglib= $(LINKDL) $<
```

Comment:
I thought the main use case is to be able to quickly debug into a failing unittest, e.g.
`make -f posix.mak std/algorithm/iteration.test` (compiles the test of said module and links them with phobos, like `.test`) and then doing sth. like:

```
b std/algorithm/iteration.test:121
run
```